google-marketplace/README.md

```diff
@@ -0,0 +1,173 @@
+# Google Cloud Launcher
+
+## Overview
+
+Datadog provides infrastructure monitoring, application performance monitoring, and log management in a single-pane-of-glass view so teams can scale rapidly and maintain operational excellence.
+
+Installing the Datadog package via Google Cloud Launcher deploys the Datadog agent on every node in your Kubernetes cluster, and configures it with a secure, RBAC-based authentication and authorization model.
+
+## Installation
+
+### Quick install with Google Cloud Marketplace
+
+Get up and running with a few clicks! Install the Datadog agent daemonset to a
+Google Kubernetes Engine cluster using Google Cloud Marketplace.
+
+Prior to that:
+
+- Create a Datadog [account](https://www.datadoghq.com/)
+- Get your Datadog API key [here](https://app.datadoghq.com/account/settings#api)
+
+Then follow the [on-screen instructions](https://console.cloud.google.com/marketplace/details/datadog-saas/datadog).
+
+### Command line instructions
+
+Follow these instructions to install the Datadog agent from the command line.
+
+#### Prerequisites (one time setup)
+
+##### Command-line tools
+
+You'll need the following tools in your development environment:
```

Comment:
r/`You'll need the following tools in your development environment:`/`Your development environment should contain the following tools:`